### PR TITLE
Merging 2 templates - adsplatform to google sheet and shopify to datastudio

### PR DESCRIPTION
**Pull request by MonikaFeigler**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+.idea
+.DS_Store
+keboola-dft-main.code-workspace
+XXX-keboola-dft-main.code-workspace
```
Comment (webrouse) [inline .gitignore:4]: Can you please remove these lines and move them to your local Git global `.gitignore`.
Those are some your tools, other people use other tools.
Thanks!

Comment (MonikaFeigler) [reply .gitignore:4]: I don't. get it I do not have it in the file localy.

Comment (webrouse) [reply .gitignore:4]: https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

File: .gitignore [hunk 2]
```diff
@@ -1,4 +1,3 @@
+/.env.local
 .idea
-.DS_Store
-keboola-dft-main.code-workspace
-XXX-keboola-dft-main.code-workspace
+.DS_Store
```
Comment (webrouse) [inline .gitignore:3]: ? :)

**Review by webrouse — APPROVED** [review]

**Pull request merged**